cleentfaar/slack-bundle/0.20/config/packages/cleentfaar_slack.yaml

```diff
@@ -0,0 +1,3 @@
+cl_slack:
+    api_token: "%env(SLACK_API_TOKEN)%"
```

Comment:
Single quotes to be consistent with the rest of yaml files?

Reply:
Ping @Devolicious. If you update this it will also run travis so we can see the issues you experience.